src/SWP/WebRendererBundle/Controller/ContentController.php

```diff
@@ -61,6 +61,10 @@ private function renderPage($type, $parameters = [])
             $context->registerMeta('article', $article);
         }
 
-        return $this->render('views/article.html.twig');
+        $tenantContext = $this->get('swp_multi_tenancy.tenant_context');
+
+        return $this->render('views/article.html.twig', [
+            'tenant' => $tenantContext->getTenant(),
```

Comment:
:+1: I will create a separate ticket. But I'm not sure if it will go into current sprint?